i18n/dumb/dumb.go

```diff
@@ -17,15 +17,8 @@
  *
  */
 
-package progress
+package i18n // import "github.com/snapcore/snapd/i18n/dumb"
```

Comment:
It seems safer to replace the real stuff under the actual i18n package for the time being. It's a very small amount of logic that may be brought back, and we'll likely want something in pure Go when we bring it back.

Not doing that will mean we'll get cgo magically re-introduced by simply screwing up a single import line on an arbitrary package.